save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPluginConfig.kt

```diff
@@ -20,7 +20,7 @@ import kotlinx.serialization.UseSerializers
  */
 @Serializable
 data class FixPluginConfig(
-    val execFlags: String,
+    val execFlags: String? = null,
     val resourceNameTestSuffix: String? = null,
```

Comment:
where are defaults for suffixes?

Reply:
Same explanation as in https://github.com/cqfn/save/pull/116#discussion_r645449191